From 2d31baa7831166fd27a2fc01ecc684613046bb28 Mon Sep 17 00:00:00 2001
From: Austin Shafer <ashafer@badland.io>
Date: Fri, 23 Aug 2024 10:12:10 -0400
Subject: [PATCH 1/9] egl-wayland bump version to 1.1.17

We are changing the release process and bumping versions immediately
after a release tag is made, that way versions installed from the
latest git master will always have the "next" version.
---
 configure.ac                       | 2 +-
 include/wayland-external-exports.h | 2 +-
 meson.build                        | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 75489a5..36594a9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@ AC_PREREQ([2.64])
 
 m4_define([wayland_eglstream_major_version], [1])
 m4_define([wayland_eglstream_minor_version], [1])
-m4_define([wayland_eglstream_micro_version], [16])
+m4_define([wayland_eglstream_micro_version], [17])
 m4_define([wayland_eglstream_version],
           [wayland_eglstream_major_version.wayland_eglstream_minor_version.wayland_eglstream_micro_version])
 
diff --git a/include/wayland-external-exports.h b/include/wayland-external-exports.h
index 43579c9..2993dea 100644
--- a/include/wayland-external-exports.h
+++ b/include/wayland-external-exports.h
@@ -53,7 +53,7 @@
  #define WAYLAND_EXTERNAL_VERSION_MINOR                      0
 #endif
 
-#define WAYLAND_EXTERNAL_VERSION_MICRO                       16
+#define WAYLAND_EXTERNAL_VERSION_MICRO                       17
 
 
 #define EGL_EXTERNAL_PLATFORM_VERSION_MAJOR WAYLAND_EXTERNAL_VERSION_MAJOR
diff --git a/meson.build b/meson.build
index 4d9bd1a..b3700fa 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('wayland-eglstream', 'c',
-        version : '1.1.16',
+        version : '1.1.17',
         default_options : [
           'buildtype=debugoptimized',
           'c_std=gnu99',
-- 
2.43.0

